src/ripple/ledger/ReadView.h

```diff
@@ -62,7 +88,13 @@ struct Fees
     XRPAmount
     accountReserve (std::size_t ownerCount) const
     {
-        return { reserve + ownerCount * increment };
+        return reserve + ownerCount * increment;
```

Comment:
This is one example of type safety lost that disappears with the new operators in `tagged_integer`. Not that it's the wrong result, it's just not obvious reading. -> `Drops32 + std::size_t * Drops32` ... size_t is basically a `uint64_t`, does `Drops32` get promoted to `Drops64`? Does `size_t` get truncated? 

If those didn't exist then we might see  `reserve + Drops32{ ownerCount } * increment` which makes it a little clearer there's a conversion happening.